doc/classes/Node3DEditorViewport.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="Node3DEditorViewport" inherits="Control" version="4.0">
+	<brief_description>
+		Control used by the 3D editor to display 3D scenes.
+	</brief_description>
+	<description>
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="can_drop_data_fw" qualifiers="const">
+			<return type="bool">
+			</return>
+			<argument index="0" name="point" type="Vector2">
+			</argument>
+			<argument index="1" name="data" type="Variant">
+			</argument>
+			<argument index="2" name="from" type="Control">
+			</argument>
+			<description>
+			</description>
+		</method>
+		<method name="drop_data_fw">
+			<return type="void">
+			</return>
+			<argument index="0" name="point" type="Vector2">
+			</argument>
+			<argument index="1" name="data" type="Variant">
+			</argument>
+			<argument index="2" name="from" type="Control">
+			</argument>
+			<description>
+			</description>
+		</method>
+		<method name="get_camera">
+			<return type="Camera3D">
+			</return>
+			<description>
+				Gets the camera used by the viewport.
+			</description>
+		</method>
+		<method name="set_message">
+			<return type="void">
+			</return>
+			<argument index="0" name="message" type="String">
+			</argument>
+			<argument index="1" name="time" type="float" default="5.0">
+			</argument>
+			<description>
+				Displays a message in the bottom corner of the viewport for a limited duration.
+			</description>
+		</method>
+		<method name="update_transform_gizmo_view">
+			<return type="void">
+			</return>
+			<description>
+				Updates the transform gizmo belonging to the viewport.
+			</description>
+		</method>
+	</methods>
+	<signals>
+		<signal name="clicked">
+			<argument index="0" name="viewport" type="Object">
+			</argument>
+			<description>
+			</description>
+		</signal>
+		<signal name="toggle_maximize_view">
+			<argument index="0" name="viewport" type="Object">
+			</argument>
+			<description>
+			</description>
```

Comment:
Please add a description here :slightly_smiling_face:

Reply:
I can take an educated guess when this signal is emitted based on the name and the source code, but I can't seem to maximize any viewports to test it! Is there a control or shortcut I don't know about?